.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+# Not adding in this PR, but I'd like to try adding a global owner set with the entire team.
+# One interpretation of their docs is that global owners are added only if not removed
+# by a more local rule.
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are order first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+# C# samples:
+/csharp/**  @BillWagner
+
+#### Snippets (under samples)
+# C++ snippets
+/snippets/cpp/** @rpetrusha
+# C# Snippets:
+/snippets/csharp/** @BillWagner
```

Comment:
this will be a lot of samples for you and Ron to review if you set the entire snippets/language to one of you

Reply:
This is true, but that's what we had in the docs repo.  

It has not been a lot of change so far. 

@rpetrusha What do you think?

Reply:
Sorry, @BillWagner. I didn't get email notification with your question. But I think that this is fine; there haven't been many changes.

Reply:
If you guys are ok with it, I'm good